ting_subsearch_suggestions.module

```diff
@@ -225,21 +254,35 @@ function ting_subsearch_suggestions_cron() {
 }
 
 /**
- * Insert kpi keys into table.
+ * Remove passed week from db.
+ *
+ * @param int $weeknr
+ *   Week number which have to be deleted.
  */
-function ting_subsearch_suggestions_insert_records($result) {
+function _ting_subsearch_suggestions_clear_passed_week($weeknr) {
+  db_delete('ting_subsearch_kpi_index_popular_keys')
+    ->condition('weekno', $weeknr, '=')
+    ->execute();
+}
```

Comment:
This function is invoked only once, so I wonder why this particular database call was wrapped into a separate function. To improve this comment, the logic where this function is called could be refactored in a way that all unnecessary records are wiped out in one call.
```
foreach (array_keys($feed_week_nrs) as $feed_week_nr) {
  if (in_array($feed_week_nr, $week_nrs_db)) {
    // Remove this week from db.
    // Collect id's that are to be wiped out.
  }
}
// Wipe out all records using the IN operator.
```